multi-agent/lib/mas/resources/repository/builtin_user_config_repository.py

```
@@ -0,0 +1,38 @@
+from abc import ABC, abstractmethod
+from typing import List, Optional
```

Ruff (0.16.0)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)